### PR TITLE
Add the governance plugin

### PR DESCRIPTION
**Pull request by ingeniumed**
This PR is meant to add the [VIP Governance Plugin](https://github.com/Automattic/vip-governance-plugin/tree/trunk) plugin to mu-plugins-ext so that it can be made available to all VIP customers, just like the Block Data API.

Corresponding mu-plugins PR can be found [here](https://github.com/Automattic/vip-go-mu-plugins/pull/4907) but this one should be merged first.

### CODE REVIEW

File: config.json
Comment (alecgeatches) [inline config.json]: Not a big deal, but you can leave `current: {}` empty and it'll get filled in automatically when it's picked up. We did the [same thing](https://github.com/Automattic/vip-go-mu-plugins-ext/pull/35/files#diff-587cb980af76fdc7e52369fd0b9d926dff266976b6f8ac631e358fecc49ff8cfR54) for the Block Data API.

**Review by WPprodigy — APPROVED** [review]

File: config.json
Comment (ingeniumed) [reply config.json]: Fixed

**Comment by ingeniumed** [comment]
I'll merge this once the other PR has been approved as well, just so they go in at the same time.

**Pull request merged**
